pkgdb2client.py

```diff
@@ -0,0 +1,25 @@
+import pkgdb2client
+import json
+
+def get_packages(name):
+	packages = []
+	pkdb = pkgdb2client.PkgDB()
+	for distro in ['f19', 'f20', 'f21', 'f22', 'f23', 'f24', 'fc25', 'el6', 'epel7']:
```

Comment:
Is there a way how to get a list of branches for a given package? Instead of hard-coding the list?